c++/src/RLEv1.cc

```diff
@@ -169,11 +169,11 @@ void RleEncoderV1::writeVslong(int64_t val) {
 
 void RleEncoderV1::writeVulong(int64_t val) {
   while (true) {
-    if ((val & ~0x7f) == 0) {
+    if ((val & ~BASE_128_MASK) == 0) {
```

Comment:
I don't think we need change here as ~0x7f will be implicitly converted to int64_t.
Same for the below.

Reply:
It is added to match code in RleDecoderV1::readLong